css/css-color/a98rgb-004-ref.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Color 4: CSS Color 4: a98-rgb</title>
+<style>
+    .test { background-color: lab(83.2141% -129.1072 87.1718); width: 12em; height: 12em; } /* color(a98-rgb 0 1 0) converted to Lab */
```

Comment:
This one is outside sRGB gamut: color(srgb -0.663902 1.00001 -0.22919) and also out of display-p3 gamut color(display-p3 -0.423708 0.979313 0.177436)

so I wonder about the pass condition; the a98-rgb and the lab should both gamut map to the same result, so the "single square" pass condition should be met.

Okay, yeah, I think this is fine.